### PR TITLE
Add localization subproject docs

### PR DESCRIPTION
**Pull request by a-mccarthy**
This PR is to add docs around the localization subproject. More context can be found in this issue: https://github.com/kubernetes/website/issues/37200

To Do:
* Reach out to localization teams for review
* Make sure all leaders are aware they are listed on this document and are still acting in this role
* This content updates/improves on the subproject information on https://github.com/kubernetes/community/tree/master/sig-docs, so we should consider some cleanup work to make sure we aren't duplicating things

**Which issue(s) this PR fixes**: 
Fixes # https://github.com/kubernetes/website/issues/37200

### CODE REVIEW

**Comment by a-mccarthy** [comment]
@divya-mohan0209 @reylejano @natalisucks If you have a chance, i would also appreciate your feedback here. thanks!

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:5]: Several nits.

```suggestion
The Kubernetes documentation has been localized into several different languages. Each localization is owned by a team of individuals that speaks the language.  A localization team is a self-sufficient community responsible for prioritizing work, translating content, and reviewing and merging PRs into the kubernetes/website repo for their localization. 
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:27]: We have this, but I don't think there is a mandate for the localization leads/contributors to join this group. I see very few members (~20) in the group below. A related topic for broader discussion is how we should be aligning the localization meeting & language-specific meeting invites to this google group so that we can ensure streamlining of comms. For sure, the former one has been created under SIG Docs official. 

Thoughts?

```suggestion
- Mailing list: https://groups.google.com/g/kubernetes-sig-docs-localization
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:44]: Nit.
```suggestion
- Making decisions for localizations policies or teams without the localization team's input
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:37]: Side Note: This is something that we should actively advocate for, as well. Despite documentation being present, there's a lot of confusion around how to start & I think we'd benefit from a maintainer track session at KubeCon EU/NA reg. this by one of the localization team maintainers.

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:38]: Link to the docs.
```suggestion
- Making sure that [Localization process documentation](https://kubernetes.io/docs/contribute/localization/) is kept up to date
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:72]:
```suggestion
Anyone can start a new localization effort for any language or join an ongoing localization team. If you are interested in learning more about specific localization requirements or processes, see the [Localization Contributing Guide](https://kubernetes.io/docs/contribute/localization/).  
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:74]:
```suggestion
Localization teams are self-sufficient communities within SIG Docs, and are responsible for
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:78]:
```suggestion
- Building a community that can support their localization efforts. Note that you need at least two contributors to begin a localization team because contributors can't approve their pull requests. You will likely need more contributors, reviewers, and approvers to create a sustainable localization effort.
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:97]: We'd probably not want all these extra lines at the end.

**Review by divya-mohan0209 — COMMENTED** [review]
Left some initial comments. Hope you find them useful!

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md:27]: This is a good point @divya-mohan0209! We have the mailing list, but we don't use it too much, except for meeting reminders. And we don't require people to join it. 

I'm really not sure if we have the need for it, but maybe we can ask for feedback to see if folks would prefer to get communication in email or slack. But if we decide to use the localization mailing list, we should make sure folks join it. 

The localization meeting invite is through the main sig docs mailing list, not the localization mailing list.

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated

**Comment by a-mccarthy** [comment]
Thanks for reviewing @divya-mohan0209!

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md:37]: huge +1 on this. Doing a maintainer track is a good idea. Is this something we can still submit for Kubecon EU in April?

File: sig-docs/localization-subproject/README.md
Comment (shurup) [inline sig-docs/localization-subproject/README.md:42]:
```suggestion
- Running localization efforts for a language
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [reply sig-docs/localization-subproject/README.md:37]: Yep, 27th January is our deadline, and I'd be happy to submit a nom for folks who are interested in presenting and have the bandwidth to do so during KubeCon

Comment (divya-mohan0209) [reply sig-docs/localization-subproject/README.md:37]: @a-mccarthy I just checked with @natalisucks and @reylejano, and they're also on board with this suggestion. However, this struck me a little late. Otherwise, we could have floated this during yesterday's meeting. WDYT about floating this news in the #sig-docs-localization channel and letting one of us know reg. the nomination?

Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md:37]: @divya-mohan0209 yes, i will ask if there is anyone interested in sharing their experiences!

File: sig-docs/localization-subproject/README.md
Comment (anubha-v-ardhan) [inline sig-docs/localization-subproject/README.md:48]:
```suggestion
- Familiar with the [localization processes](https://kubernetes.io/docs/contribute/localization/).
```

Comment (anubha-v-ardhan) [reply sig-docs/localization-subproject/README.md:48]: Or: `Familiarity with the localization processes`

File: sig-docs/localization-subproject/README.md
Comment (anubha-v-ardhan) [inline sig-docs/localization-subproject/README.md:53]:
```suggestion
Time commitment for a Localization lead is 1-2 hours per week.
```

**Review by stormqueen1990 — COMMENTED** [review]
Found a few minor typos!

**Comment by a-mccarthy** [comment]
@stormqueen1990 @anubha-v-ardhan and @shurup thank you for the reviews! I've hopefully corrected all the items you called out.

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:3]: nit.
```suggestion
The Kubernetes Localization Subproject is owned by [SIG Docs](https://github.com/kubernetes/community/tree/master/sig-docs) and run by the [Subproject leads](#leadership).
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:11]:
```suggestion
Regular Localization Meeting: The first Monday of the month at 15:00 UTC (biweekly). [Convert it to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC)
```

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [inline sig-docs/localization-subproject/README.md:36]:
```suggestion
- Advocating for localization-friendly practices within SIG Docs and the broader Kubernetes community.
```

**Review by divya-mohan0209 — COMMENTED** [review]
Few nits. But overall LGTM! Thank you once again, Abbie :)

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:42]:
```suggestion
- Running localization efforts for a language.
```

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:43]:
```suggestion
- Dictating how localization teams do work or organize.
```

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:44]:
```suggestion
- Making decisions for localizations policies or teams without the localization team's input.
```

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:50]:
```suggestion
- A working knowledge of Github.
```

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:46]: IMHO it does not make sense to have 3rd heading section, if there is only one on that level. 
```suggestion
## Skills and Experience Required 
```

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:55]:
```suggestion
## Localization teams
```
First level Heading should only be used for the Headline of the whole doc IMHO.

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:79]:
```suggestion
### Localization team structure
```

File: sig-docs/localization-subproject/README.md
Comment (iamNoah1) [inline sig-docs/localization-subproject/README.md:86]:
```suggestion
- Timely reviews and merging PRs for their languages according to the [review and merge processes](https://kubernetes.io/docs/contribute/review/) of the kubernetes/website repo.
```

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
Left some suggestions/corrections.

File: sig-docs/localization-subproject/README.md
Comment (natalisucks) [inline sig-docs/localization-subproject/README.md:11]: The addition of biweekly here confuses me, since this is a monthly meeting – am I interpreting this incorrectly?

Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: updated!

**Comment by natalisucks** [comment]
A big thanks to the community for reviewing this, and a huge thank you to @a-mccarthy for this work! 🎉 
/lgtm

File: sig-docs/localization-subproject/README.md
Comment (kbhawkey) [inline sig-docs/localization-subproject/README.md:5]: Some possible edits:
The Kubernetes documentation is localized into several [or many] different languages.
Each localization is owned by a team of contributors that specialize in translating [localizing] the Kubernetes docs into a specific language.

File: sig-docs/localization-subproject/README.md
Comment (kbhawkey) [inline sig-docs/localization-subproject/README.md:50]: nit: GitHub

**Comment by shurup** [comment]
Hi everyone! I have been looking for this doc a couple of times already, and it was challenging to find it due to not being merged yet. 

Are there any plans to move forward with merging this PR? Are there any show-stoppers for that? Can I help?

**Comment by a-mccarthy** [comment]
@shurup, yes there are plans to move forward with this PR! Sorry, for it being delayed, I had to take some time away and didn't realize this was still open. 

As far as blockers, i don't believe there are any. The only piece that might need another review is the list of localization team leaders. I'd like to make sure everyone is still willing to be in that role and make sure we aren't missing anyone

**Comment by a-mccarthy** [comment]
@seokho-son, do you have some time to review this doc on the localization subproject? Would be great to get your feedback and 👍

File: sig-docs/localization-subproject/README.md
Comment (seokho-son) [inline sig-docs/localization-subproject/README.md:102]:
```suggestion
* Experience reviewing English language PRs to develop an understanding the review processes and repo structure. It is recommended that leaders participate in the SIG Docs [PR Wrangler Shadow Program](https://kubernetes.io/docs/contribute/participate/pr-wranglers/#pr-wrangler-shadow-program).
```

**Review by seokho-son — COMMENTED** [review]
Hi @a-mccarthy 
I've reviewed and left some suggestions. :)

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md:97]: >Do we need to guide `It's recommended that each localization team has at least 2 leads.` ?
(according to the above current list, only few teams satisfy the recommendation)

I think its still good to recommend that there are at least 2 people as leads, even if we aren't able to staff it. I think its good to still advocate for having more leaders because we don't want to have a single point of failure for a team if someone needs to step away from the project and a gentle nudge for leaders to keep looking for more folks to help them do the work Its worded as a "recommendation" and not a requirement for a localization team thought because its not always possible or available to folks. 

> we can make the team lead as an official role
I'd like to make the team lead more of an official role within the project so that folks can get the proper recognition. I don't know the process for formalizing a role though in the community but I think if this was to be a more formal role, folks who wanted to be a lead would need to go through the a nomination/approval process like in the other roles within the community.

File: sig-docs/localization-subproject/README.md
Comment (a-mccarthy) [reply sig-docs/localization-subproject/README.md]: @seokho-son I updated the table to align with your suggestion. Great idea! Can you take another look?

File: sig-docs/localization-subproject/README.md
Comment (divya-mohan0209) [reply sig-docs/localization-subproject/README.md:97]: @seokho-son & @a-mccarthy : 

- Recommending 2 people as leads here is appropriate because it is not just a recommendation. At the moment, it is a requirement for new localisations going forward so that we can ensure consistency wrt contributions and not just when it goes live.
- Acknowledging team leads/localization maintainers is an idea I'm 100% on board with. However, these are the challenges I foresee with making that a reality, 
  - Processes aren't uniform across all localization. There will be concerns around the implementation if we go forward without achieving uniformity there first.
  - We'll need to address other major questions that come with the formalisation of a role like - maintaining staffing, a consistent contributor pool, & ensuring necessary diversity across the board. For some localisations that are relatively smaller, this might not be a discussion we can actually have.
  - Additionally, formalised roles would typically go through the nomination/approval process, seeking community consensus and would significantly increase the barrier to getting localisations started and live.

File: sig-docs/localization-subproject/README.md
Comment (seokho-son) [reply sig-docs/localization-subproject/README.md]: @a-mccarthy Looks good to me ! :)

File: sig-docs/localization-subproject/README.md
Comment (seokho-son) [reply sig-docs/localization-subproject/README.md:97]: Hi @a-mccarthy @divya-mohan0209,
I concur with the recommendations outlined in the section. Additionally, I acknowledge @divya-mohan0209 's concerns regarding the formalization of roles. My suggestion is that we move forward with the current terminology as it stands, specifically the recommendation that `each localization team should have at least two leads`. Subsequently, we can assess the practicality of formalizing these roles across the existing teams.

**Comment by seokho-son** [comment]
LGTM

**Comment by natalisucks** [comment]
/approve
I'd like to see us getting this live ASAP and revising further as needed. Let's get an LGTM from the community and move this forward for the sake of the subproject. Once again, a huge thank you to @a-mccarthy, @seokho-son, and the rest of the community for their great efforts ❤️

**Review by sftim — COMMENTED** [review]
nit and comments from https://github.com/kubernetes/community/pull/7026/files#r1323192899 noted

/lgtm
Merging this change will be an improvement.

**Pull request merged**
